pwchem/protocols/VirtualDrugScreening/protocol_ligand_descriptor_calc.py

```diff
@@ -0,0 +1,144 @@
+# **************************************************************************
+# *
+# * Authors: ...
+# *
```

Comment:
This protocol is redundant right? I am guessing it was the first draft and then you changed the name but did not remove this